Maestro/subscriptions.json

```diff
@@ -646,8 +646,15 @@
         "https://github.com/dotnet/arcade/blob/release/**/*",
         "https://github.com/dotnet/diagnostics/blob/master/**/*",
         "https://github.com/dotnet/diagnostics/blob/release/**/*",
+        "https://github.com/dotnet/core-sdk/blob/master/**/*",
```

Comment:
I'm going to go ahead and merge this, might be nice to keep these trigger lists alphabetized in the future though.